docs: update app installation steps with accelerator resources selection (#3392)
beclab/Olares: docs/manual/olares/market/market.md

To install an application from Market:

1. Open Market from the Dock or Launchpad.
2. Find your target application, and double-click it to view its details.

> 3. If the application supports multiple hardware accelerators, configure your deployment in the **RESOURCES** section:

Reviewer: Would it be better to add a screenshot here to show the **RESOURCES** section？

Author: yes will add one when the ui is ready

    a. Select your preferred computing resource from the drop-down list. For example, **NVIDIA GPU**, **NVIDIA GB10**, or **CPU**.

    <!--![Accelerator resources](/images/manual/olares/market-accelerator.png#bordered)-->

>     b. Review the updated **CPU**, **Memory**, **Required disk**, and **VRAM** requirements, and make sure your hardware meets these requirements.

Reviewer: ```suggestion
    b. Review the **CPU**, **Memory**, **Required disk**, and **VRAM** requirements for the selected computing resource, and make sure your hardware meets them.
```

## FAQs

### Errors in the RESOURCES section

Before you install an application, the system evaluates your current hardware and cluster capacity. If your environment does not meet the selected configuration, an inline error message will appear directly inside the **RESOURCES** panel:

- "Unable to install. The Olares cluster does not have any required accelerator specified."

    The application requires specific computing hardware that is not installed or detected on your device or cluster. When this happens, the resource drop-down will display **N/A**, and the unsupported hardware options will be greyed out and unselectable.

- "Unable to install on this accelerator type. Not enough required {Resource} on Olares cluster."

>     Your system possesses the correct hardware type (e.g., an NVIDIA GPU), but its currently available capacity (such as GPU memory) is insufficient to run the app. You must stop other running applications to free up the required resources for this specific accelerator.

Reviewer: ```suggestion
    Your device or cluster has the required accelerator type, but the selected accelerator does not have enough available resources, such as GPU memory. You must stop other running applications to free up the required resources for this specific accelerator.
```

- "Unable to install. Not enough required {Resource} available on your quota."

>     Your personal account's assigned resource limit (e.g., CPU or Memory) has been reached. Try freeing up your allocated resources by stopping your unused apps, or contact your administrator to request a quota increase.

Reviewer: ```suggestion
    Your account has reached its assigned resource quota, such as CPU or memory. Try freeing up your allocated resources by stopping your unused apps, or contact your administrator to request a quota increase.
```
